src/Pyz/Yves/ShopUi/Theme/default/styles/settings/_color.scss

```
@@ -10,12 +10,12 @@ $setting-color-gray-3: #8f8f8f;
 $setting-color-gray-4: #4c4c4c;
 $setting-color-gray-5: #333;
 $setting-color-gray-6: #e7eaee;
-$setting-color-gray-7: #f0f0f0;
+$setting-color-gray-7: #f6f6f6;
 
-$setting-color-blue-1: #0064b4;
+$setting-color-blue-1: #005ba1;
 
-$setting-color-yellow-1: #fdbe36;
-$setting-color-yellow-2: #fdcb5e;
+$setting-color-yellow-1: #d88d00;
+$setting-color-yellow-2: #d88d00;
```

Why do we need two variables with the same value?

---

Previous colors were not contrasted enough so I changed the color for them. Two variables with the same color code because we should not remove what already exists.

---

In this case I'd suggest the following and replace all usages of `$setting-color-yellow-2` with `$setting-color-yellow-1` so we have just one source of truth.

```suggestion
$setting-color-yellow-1: #d88d00;
$setting-color-yellow-2: $setting-color-yellow-1;
```

---

From one perspective it's a project level, from another - it will e much easier to integrate these changes. We can discuss it.

---

Changed value